src/MainWindow.cpp

```diff
@@ -492,6 +492,14 @@ MainWindow::MainWindow(WSClient *client, DbMasterController *mc, QWidget *parent
     connect(ui->checkBoxBoot, SIGNAL(toggled(bool)), this, SLOT(checkSettingsChanged()));
     connect(ui->checkBoxTuto, SIGNAL(toggled(bool)), this, SLOT(checkSettingsChanged()));
 
+#ifdef Q_OS_LINUX
+    ui->checkBoxLockDevice->setChecked(false);
+    ui->checkBoxLockDevice->setEnabled(false);
```

Comment:
I would prefer to hide the checkbox on linux for now

Reply:
I can do that.